src/io/invoca/Docker.groovy

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/groovy
+package io.invoca;
+
+/* args
+    build_args  list (optional)
+    dockerfile  string
+    image_name  string
+*/
+
+@NonCPS
+def buildCommand(Map args) {
+    def String build_args = ["", *args.build_args].join(' --build-arg ')
```

Comment:
This is a bit confusing, maybe move it down inside the block that appends the build args to the command?  You could then do away with the local variable and the empty string join.